_extras/guide.md

```diff
@@ -3,293 +3,548 @@ layout: page
-    *   Clone your repo in a different directory, pretending is your computer at
-        work: `git clone https://github.com/vlad/planets.git planets-at-work`.
-    *   At the office, you make a change, commit and push.
-    *   At your laptop repo, you (forget to pull and) make a change, commit and
-        try to push.
-    *   `git pull` now and show the conflict.
-
-*   Learners usually forget to `git add` the file after fixing the conflict and
-    just (try to) commit. You can diagnose this with `git status`.
-
-*   Remember that you can discard one of the two parents of the merge:
-
-    *   discard the remote file, `git checkout --ours conflicted_file.txt`
-    *   discard the local file, `git checkout --theirs conflicted_file.txt`
-
-    You still have to `git add` and `git commit` after this. This is
-    particularly useful when working with binary files.
-
-## [Open Science]({{ page.root }}/10-open/)
-
-## [Licensing]({{ page.root }}/11-licensing/)
-
-We teach about licensing because questions about who owns what, or can use
-what, arise naturally once we start talking about using public services like
-GitHub to store files. Also, the discussion gives learners a chance to catch
-their breath after what is often a frustrating couple of hours.
-
-## [Citation]({{ page.root }}/12-citation/)
-
-## [Hosting]({{ page.root }}/13-hosting/)
-
-A common concern for learners is having their work publicly available on
-GitHub.  While we encourage open science, sometimes private repos are the
-only choice. It's always interesting to mention the options to have
-web-hosted private repositories.
-
-[code-school]: https://www.codeschool.com/
-[diffmerge]: https://sourcegear.com/diffmerge/
-[drawings]: https://marklodato.github.io/visual-git-guide/index-en.html
-[git-it]: https://github.com/jlord/git-it
-[git-it-electron]: https://github.com/jlord/git-it-electron
-[git-parable]: http://tom.preston-werner.com/2009/05/19/the-git-parable.html
-[github]: https://github.com/
-[github-gui]: http://git-scm.com/downloads/guis
-[github-line-endings]: https://help.github.com/articles/dealing-with-line-endings/#platform-all
-[github-privacy]: https://help.github.com/articles/keeping-your-email-address-private/
-[repos-in-repos]: https://github.com/swcarpentry/git-novice/issues/272
-[try-git]: https://try.github.io
+### Section Overview:
+- "Where does Git store information?"
+- "How do I record changes in Git?"
+- "How do I check the status of my version control repository?"
+- "How do I record notes about what changes I made and why?"
+
+### Instructions:
+- Change to `cats-as-data` directory
+~~~
+$ cd ~/Desktop/cats-as-data # or wherever you unzipped it
+~~~
+{: .bash}
+
+- **New Command**: `git init` to initialize our git repository
+~~~
+$ git init
+~~~
+{: .bash}
+
+- If we list the files, we can not see a hidden `.git` directory. Git stores all information about our project in here.
+  If we ever delete it, we will lose our project history
+~~~
+$ ls -a
```

Comment:
Do you want to add `dir /ah` for Windows?

Reply:
If they're using the Git Bash thing for Windows, do they need `dir` still, or do they have `ls`?